tests/php/Collaboration/Collaborators/RoomPluginTest.php

```diff
@@ -34,22 +35,16 @@
 use OCP\IUserSession;
 use OCP\Share\IShare;
 
-class RoomPluginTest extends \Test\TestCase {
+class RoomPluginTest extends TestCase {
+	protected ?Manager $manager = null;
 
-	/** @var Manager */
-	protected $manager;
+	protected ?IUserSession $userSession = null;
```

Comment:
this is also a MockObject  but it's not documented as such so I guess some IDEs will fail to offer autocomplete